core/modules/simpletest/tests/common.test

```diff
@@ -945,15 +945,15 @@ class CommonHTMLIdentifierTestCase extends BackdropUnitTestCase {
    */
   function testBackdropCleanCSSIdentifier() {
     // Verify that no valid ASCII characters are stripped from the identifier.
-    $identifier = 'abcdefghijklmnopqrstuvwxyz_ABCDEFGHIJKLMNOPQRSTUVWXYZ-0123456789';
+    $identifier = 'abcdefghijklmnopqrstuvwxyz_ABCDEFGHIJKLMNOPQRSTUVWXYZ-./0123456789:@';
```

Comment:
By default, we won't change this line. Those characters still get stripped out. But they can be allowed by altering the `$filter` in the alter hook.